### PR TITLE
doc(readme): detail permissions per probe + update global monitor profile configuration

### PR DESCRIPTION
**Pull request by IMSdevsecu**
Allows for granular permission granting according to selected category probe name.
Also fixing bad configuration or deprecated ones in the global monitor profile configuration.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -290,36 +290,71 @@ To probe a FortiGate, do something like `curl 'localhost:9710/probe?target=https
 
 The following example Admin Profile describes the permissions that needs to be granted
 to the monitor user in order for all metrics to be available.
```
Comment (bluecmd) [inline README.md:292]: Let's move these two lines to after the table.

Comment (IMSdevsecu) [reply README.md:292]:
```suggestion
```

File: README.md [hunk 2]
```diff
@@ -290,36 +290,71 @@ To probe a FortiGate, do something like `curl 'localhost:9710/probe?target=https
...
+|VPN/Ssl/Connections          | vpngrp             |api/v2/monitor/vpn/ssl |
+|VPN/Ssl/Stats                | vpngrp             |api/v2/monitor/vpn/ssl/stats |
+|VirtualWAN/HealthCheck       | netgrp.cfg         |api/v2/monitor/virtual-wan/health-check |
+|Wifi/APStatus                | wifi               |api/v2/monitor/wifi/ap_status |
+|Wifi/Clients                 | wifi               |api/v2/monitor/wifi/client |
+|Wifi/ManagedAP               | wifi               |api/v2/monitor/wifi/managed_ap |
+
 If you omit to grant some of these permissions you will receive log messages warning about
 403 errors and relevant metrics will be unavailable, but other metrics will still work.
+If you do not need some probes to be run, do not grant permission for them and use `include/exclude` feature (see `Usage` section).
```
Comment (IMSdevsecu) [inline README.md:328]:
```suggestion

The following example Admin Profile describes the permissions that needs to be granted
to the monitor user in order for all metrics to be available.

```

**Review by bluecmd — APPROVED** [review]

**Pull request merged**